trackers/core/reid/dataset/base.py

```diff
@@ -38,7 +44,7 @@ def __init__(
         transforms: Optional[Compose] = None,
     ):
         self.tracker_id_to_images = validate_tracker_id_to_images(tracker_id_to_images)
-        self.transforms = transforms or ToTensor()
+        self.transforms: Union[Compose, ToTensor] = transforms or ToTensor()
```

Comment:
```suggestion
        self.transforms: Compose = transforms or transforms.Compose([ToTensor()])
```

@soumik12345 would it make sense to update this line like this? It might help unify the MyPy typing in this part of the code.

Reply:
I think a better type hint would be `Union[Compose, Callable]` because any stateful callables should technically be possible to be composed as transforms.